media-api/app/controllers/MediaApi.scala

```diff
@@ -70,9 +70,12 @@ class MediaApi(
 
     val userCanUpload: Boolean = authorisation.hasPermissionTo(UploadImages)(user)
     val userCanArchive: Boolean = authorisation.hasPermissionTo(ArchiveImages)(user)
+    val userCanViewDeletedImages: Boolean = authorisation.hasPermissionTo(ViewDeletedImages)(user)
+
 
     val maybeLoaderLink: Option[Link] = Some(Link("loader", config.loaderUri)).filter(_ => userCanUpload)
     val maybeArchiveLink: Option[Link] = Some(Link("archive", s"${config.metadataUri}/metadata/{id}/archived")).filter(_ => userCanArchive)
+    val maybeViewDeletedLink: Option[Link] = Some(Link("archive", s"${config.authUri}/{id}/hide")).filter(_ => userCanViewDeletedImages)
```

Comment:
Should this be `"archive"` here? I think it'd want to match the new route added below?